code/modules/research/designs/mechfabricator_designs.dm

```diff
@@ -1094,7 +1094,7 @@
 	name = "Cyborg Upgrade (Safety Override)"
 	id = "borg_syndicate_module"
 	build_type = MECHFAB
-	req_tech = list("combat" = 4, "syndicate" = 2)
+	req_tech = list("combat" = 7, "materials" = 7)
```

Comment:
On the one hand, I do feel like it requiring syndicate 2, despite not being syndi tech, or even a syndi item, is a good change.
On the other hand, I feel like data/programming tech might be a better requirement than materials tech.
I also wonder whether requiring two level 7 techs will make this harder, or easier, to get than requiring combat4/syndi2, and whether that's a good thing or not.

Reply:
It should be gotten at roughly the same step, as to get combat 7 you need either a combat shotgun, captains rapier or chain of command, to get past combat 5, a
It should be slightly easier then it currently, but only due to the nature of syndicate tech being locked behind either thermals implant or xray lasers gun, with thermals needing 6 mat, 4 programming, 7 bio, 5 magnets and plasma 4, and xray needing combat 7, magnets 5, bio 5 and power 5, while changing materials to data might make a bit more sense, it would make it a fair more 'difficult' to get due to the nature of only a handful of miners knowing what tech they need to bring back, which often makes it one of the missing techs.  
Again this can differ from round to round, but for your average highpop round, it will be available at roughly the same time in general, all depending on how early roboticist offers upgrades or borgs asking for them.

Reply:
Do combat 7 + data 7.

Reply:
Done